_data/tagging-status.yml

```diff
@@ -476,9 +477,9 @@
   supported-through: [phase-III,math]
   comments: "Use of math tagging currently requires support from external tools."
```

Comment:
is that statement (here and elsewhere) still true this way, or would it need to get adjusted? With luamml it should work, without any external tools, or not? @davidcarlisle ?

Reply:
From my limited understanding I'd think it can be removed but I'll wait for others' opinions